process/transaction/shardProcess.go

```diff
@@ -655,6 +698,40 @@ func (txProc *txProcessor) processTxAtRelayer(
 	return txHash, nil
 }
 
+func (txProc *txProcessor) processTxV3AtRelayer(
+	relayerAcnt state.UserAccountHandler,
+	totalFee *big.Int,
+	relayerFee *big.Int,
+	tx *transaction.Transaction,
+	valueToSubFromRelayer *big.Int,
+) ([]byte, error) {
+	txHash, err := core.CalculateHash(txProc.marshalizer, txProc.hasher, tx)
+	if err != nil {
+		return nil, err
+	}
+
+	if !check.IfNil(relayerAcnt) {
+		err = relayerAcnt.SubFromBalance(valueToSubFromRelayer)
+		if err != nil {
+			return nil, err
+		}
+
+		err = relayerAcnt.SubFromBalance(totalFee)
+		if err != nil {
+			return nil, err
+		}
+
+		err = txProc.accounts.SaveAccount(relayerAcnt)
+		if err != nil {
+			return nil, err
+		}
+
+		txProc.txFeeHandler.ProcessTransactionFee(relayerFee, big.NewInt(0), txHash)
```

Comment:
isn't the fee the Total Fee in this case ?

Reply:
I think it should stay relayerFee, same as relayed v1 and v2, as the cost of execution is added to txFeeHandler as part of sc processor